Sources/CarPlay/OACarPlayDashboardInterfaceController.mm

```diff
@@ -612,6 +612,18 @@ - (void)onMapViewAttached
     if (route && start && _routingHelper.isRouteCalculated)
     {
         [self enterRoutePreviewMode];
+        if ([_routingHelper isFollowingMode])
+            [self onTripStartTriggered];
+    }
+}
+
+- (void)onMapViewDettached
+{
+    [NSNotificationCenter.defaultCenter removeObserver:self];
```

Comment:
Why notifications and location observer are not created at `onMapViewAttached` in this case?
Remove `dealloc` method